backend/tests/agents/report_agent_test.py

```diff
@@ -14,6 +15,9 @@ async def test_invoke_calls_llm(mocker):
 
     mock_llm.chat = mocker.AsyncMock(return_value=mock_response)
 
-    response = await report_agent.create_report("Test Document", materiality_topics={"abc": "123"})
+    response = await report_agent.create_report(
+        LLMFile(file_name="test", file=b"Sample text content"),
+        materiality_topics={"abc": "123"}
+        )
```

Comment:
Might just be GitHub but it looks like `)` should be shifted left one tab

Reply:
I've deleted this file, the report agent doesn't do anything that needs testing